### PR TITLE
Add Kotlin template

### PR DESCRIPTION
**Pull request by k33g**

### CODE REVIEW

**Comment by danielpetisme** [comment]
Thanks for the PR!

Could you also provide the appropriate `pom.xml` and `build.gradle` to run the project out of the box.

W should be able to run `mvn clean compile` and the same for gradle straight from the generated project.

**Comment by danielpetisme** [comment]
As ask via #33 the project formatting required to be aligned with Vert.x standards #38 fix this issue. Please consider to fetch the updates before moving forward on your PR.
Thanks.

**Comment by k33g** [comment]
@danielpetisme 
- Maven ok
- Gradle 🚧  (I need to learn Gradle)

**remark**: there is no way to choose the Kotlin version (I set in generated `pom.xml` and `build.gradle`)

**Comment by danielpetisme** [comment]
I would rather keep the starter opinionated so let's use the latest community-proven Kotlin version.
I have in minde to make a more advanced page with much more tweaking options.
Another example is the Java version for instance. Right now the sources are flagged as Java 8 until Java 9 will be the community leader version.

Could you track the comment in an issue please to feed the V2 😉

**Comment by danielpetisme** [comment]
@spodin it seems you're used to Gradle maybe you could help or give some pointers? Thanks

**Comment by spodin** [comment]
@danielpetisme Unfortunately, currently I'm not familiar with Kotlin. I would recommend to review Gradle build file for Java project (see #37) and extend it with Kotlin required things (`compile` scoped dependencies, plugins etc.)

Also, I'm pointing to simplify main verticle template for Kotlin lang:

- Remove `main` function
- Hardcode HTTP server port to `8080` as in Java template rather than retrieving it from system environment variable

**Comment by k33g** [comment]
hey 👋 @danielpetisme @spodin 
- I will investigate about Gradle
- about `System.getenv("PORT")?.toInt() ?: 8080` , well, `8080` is hardcoded if I can't find the environment variable (even with a sample, I think it's useful for beginners, generally, it is a good practice, especially for deployment. Let's give a good example from the beginning 😉
- oups right about the `main` function

**Comment by danielpetisme** [comment]
TL;DR: 👍  to hardcode 8080

IMHO, in a first version we should provide dummy projects. I don't like to compare but for me is like `start.spring.io` and Jhipster. The first one is a way to encourage project creation and setting up the spring dependencies the other one is here to enhance the dev experience. 

In the Vert.x ecosystem, you'll find many ways to do the same thing (for instance I use a config file to provide HTTP port rather than environnement variables) let's keep things simple. 

I would prefer to have basic templates first and then think on how we could provide more elaborate project (with environment variables retrieval, DevOps dependencies, etc.).

**Comment by jponge** [comment]
https://github.com/jponge/vertx-gradle-plugin

**Comment by danielpetisme** [comment]
@k33g @jponge i've deployed a new version of the starter including a kotlin sample.
Could you test it please.

**Comment by danielpetisme** [comment]
Fixed via #57
